<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTPlayerClass</class>
 <widget class="QWidget" name="QTPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#openButton:!hover
{
    background-color: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(120, 120, 120, 255),stop:0.495      rgba(39, 39, 39, 255),stop:0.505 rgba(39, 39, 39, 255),stop:1 rgba(120, 120, 120, 255));
    border:none;
    border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 12pt &quot;黑体&quot;;
}
#openButton:hover
{
    background-color: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(39, 39, 39, 255),stop:0.495      rgba(120, 120, 120, 255),stop:0.505 rgba(120, 120, 120, 255),stop:1 rgba(39, 39, 39, 255));
    border:none;
    border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 12pt &quot;黑体&quot;;
}
#playButton{background-color: rgba(255, 255, 255,0);}

/*#playButton:hover{image: url(:/QTPlayer/Resources/PlayN.png);}

#playButton:!hover{image: url(:/QTPlayer/Resources/PlayH.png);}

#playButton:pressed{image: url(:/QTPlayer/Resources/PlayP.png);}
*/
 

#bottemWidget{
background-color: rgba(0, 0, 0, 122);
}

#totalHour{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#totalMinute{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#totalSecond{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#currentHour{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#currentMinute{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#currentSecond{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}</string>
  </property>
  <widget class="VideoOpenGLWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="bottemWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>800</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QSlider" name="playslider">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>601</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="currentContainer" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>29</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="currentLayout">
      <item>
       <widget class="QLabel" name="currentHour">
        <property name="text">
         <string>0:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentMinute">
        <property name="text">
         <string>0:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentSecond">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="totalContainer" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="totalLayout">
      <item>
       <widget class="QLabel" name="totalHour">
        <property name="text">
         <string>0:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalMinute">
        <property name="text">
         <string>0:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalSecond">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videoopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QTPlayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>QTPlayerClass</receiver>
   <slot>openVideoFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>-22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playslider</sender>
   <signal>sliderPressed()</signal>
   <receiver>QTPlayerClass</receiver>
   <slot>sliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playslider</sender>
   <signal>sliderReleased()</signal>
   <receiver>QTPlayerClass</receiver>
   <slot>sliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>707</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>QTPlayerClass</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>QTPlayerClass</receiver>
   <slot>setVolume(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openVideoFile()</slot>
  <slot>sliderPress()</slot>
  <slot>sliderRelease()</slot>
  <slot>setPos(int)</slot>
  <slot>play()</slot>
  <slot>setVolume(int)</slot>
  <slot>mirrorLeftAndRight()</slot>
  <slot>mirrorUpAndDown()</slot>
 </slots>
</ui>
